electrostatics.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+#Constants
+epsilon0 = 8.854187817e-12  #Farads per meter
+
+#Lambda function to calculate the distance between an observation point x and all charges p
+r_xp = lambda x, p: np.linalg.norm(x - p, axis=1)
```

Comment:
Agree with the linter here. Not useful use of lambda function. Also it doesn’t fulfill the homework requirement of using your ED function as an argument in another function